### PR TITLE
Document canCreate prop of EditableList

### PR DESCRIPTION
**Pull request by skomorokh**
It is in fact possible to disable record creation on this component, but that functionality was undocumented... until now.

### CODE REVIEW

File: lib/EditableList/readme.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Use the `EditableList` component in your jsx
 ### Configuration (props)
 Name | type | description | default | required
 --- | --- | --- | --- | ---
+canCreate | boolean | Determines if the '+ New' button is active. | | no
```
Comment (MikeTaylor) [inline lib/EditableList/readme.md:34]: What about setting the "default" column to `false`?

**Review by MikeTaylor — APPROVED** [review]
Thanks for this good-citzen moment.

File: lib/EditableList/readme.md (hunk 1 above)
Comment (skomorokh) [reply lib/EditableList/readme.md:34]: I'm pretty sure it's undefined if not present so only falsy not false and that column reflects that by being empty for many optional properties.

Comment (skomorokh) [reply lib/EditableList/readme.md:34]: ...hm, but actually it's true now that I see it in the table. Another PR I guess....

Comment (MikeTaylor) [reply lib/EditableList/readme.md:34]: Leave it how it is — your previous argument was convincing.

Comment (skomorokh) [reply lib/EditableList/readme.md:34]: Except that I was confused while making that previous argument and it turns out to be explicitly defaulted to true so it was most honest to be explicit about that default and indeed consistent with another column further down documented as true. So I'd gone ahead with another PR to add that.

Comment (MikeTaylor) [reply lib/EditableList/readme.md:34]: Yep, saw it, it's good.

**Pull request merged**